project_national/fact_sheets_category_4.yml

```diff
@@ -0,0 +1,318 @@
+            - HVAC Cooling Efficiency|AC, SEER 13
+            - HVAC Cooling Efficiency|AC, SEER 15
+
+  - upgrade_name: HPWH
+    options:
+      - &water_heater_fuel_type
+        option: Water Heater Fuel|Electricity
+        apply_logic:
+          - not: Water Heater Fuel|Electricity
+      - &water_heater_efficiency_electric_heat_pump_50_gal_3_pt_45_uef
+        option: Water Heater Efficiency|Electric Heat Pump, 50 gal, 3.45 UEF
+        apply_logic:
+          - and:
+            - not: Water Heater Efficiency|Electric Heat Pump, 50 gal, 3.45 UEF
+            - or:
+              - Bedrooms|1
+              - Bedrooms|2
+              - Bedrooms|3
+      - &water_heater_efficiency_electric_heat_pump_66_gal_3_pt_35_uef
+        option: Water Heater Efficiency|Electric Heat Pump, 66 gal, 3.35 UEF
+        apply_logic:
+          - and:
+            - not: Water Heater Efficiency|Electric Heat Pump, 66 gal, 3.35 UEF
+            - Bedrooms|4
+      - &water_heater_efficiency_electric_heat_pump_80_gal_3_pt_45_uef
+        option: Water Heater Efficiency|Electric Heat Pump, 80 gal, 3.45 UEF
+        apply_logic:
+          - and:
+            - not: Water Heater Efficiency|Electric Heat Pump, 80 gal, 3.45 UEF
+            - Bedrooms|5
+
+  - upgrade_name: 96% AFUE Natural Gas Furnace
+    options:
+      - &Furnace_fuel_type
+        option: HVAC Heating Type And Fuel|Natural Gas Fuel Furnace
+      - &96pct_AFUE_furnace
+        option: HVAC Heating Efficiency|Fuel Furnace, 96% AFUE
+        apply_logic:
+          - or:
+            - HVAC Heating Efficiency|Fuel Furnace, 60% AFUE
+            - HVAC Heating Efficiency|Fuel Furnace, 76% AFUE
+            - HVAC Heating Efficiency|Fuel Furnace, 80% AFUE
+            - HVAC Heating Efficiency|Fuel Furnace, 92.5% AFUE
+    package_apply_logic: 
+      - and: 
+        - or:
+          - Heating Fuel|Natural Gas
+          - Water Heater Fuel|Natural Gas
+          - Clothes Dryer|Gas
+          - Misc Hot Tub Spa|Natural Gas
+          - Misc Pool Heater|Natural Gas
+          - Misc Gas Fireplace|Gas Fireplace
+          - Misc Gas Grill|Gas Grill
+          - Misc Gas Lighting|Gas Lighting
+        - or:
+          - HVAC Heating Type And Fuel|Fuel Oil Fuel Furnace
+          - HVAC Heating Type And Fuel|Propane Fuel Furnace
+          - HVAC Heating Type And Fuel|Other Fuel Fuel Furnace
+          - HVAC Heating Type And Fuel|Natural Gas Fuel Furnace
+
+  - upgrade_name: 96% AFUE Natural Gas Boiler
+    options:
+      - &Boiler_fuel_type
+        option: HVAC Heating Type And Fuel|Natural Gas Fuel Boiler
+      - &96pct_AFUE_boiler
+        option: HVAC Heating Efficiency|Fuel Boiler, 96% AFUE
+        apply_logic:
+          - or:
+            - HVAC Heating Efficiency|Fuel Boiler, 76% AFUE
+            - HVAC Heating Efficiency|Fuel Boiler, 80% AFUE
+            - HVAC Heating Efficiency|Fuel Boiler, 90% AFUE
+    package_apply_logic: 
+      - and: 
```

Comment:
no need for `- and:` here and personal preference (for me) is to have `package_apply_logic:` at the top of the upgrade, just under `options:`